test/integration/scheduler_perf/util.go

```diff
@@ -70,10 +85,18 @@ func mustSetupScheduler() (util.ShutdownFunc, coreinformers.PodInformer, clients
 		Burst:         5000,
 	}
 
+	// use default component config if cfg is nil
+	if config == nil {
+		config, err = newDefaultComponentConfig()
+		if err != nil {
+			klog.Fatalf("Error creating default component config: %v", err)
+		}
+	}
+
 	client := clientset.NewForConfigOrDie(cfg)
 	dynClient := dynamic.NewForConfigOrDie(cfg)
 
-	_, podInformer, schedulerShutdown := util.StartScheduler(client)
+	_, podInformer, schedulerShutdown := util.StartScheduler(client, config)
```

Comment:
It seems we do not fully respect `config` - we're just using partial fields related to scheduler. Let's add a comment here.

Reply:
true, will do.